apps/main/src/dex/components/PagePool/Swap/index.tsx

```diff
@@ -545,9 +550,13 @@ const Swap = ({
           </div>
         ) : (
           <LargeTokenInput
+            label={t`Buy`}
             dataType="decimal"
             name="toAmount"
             onBalance={setToAmount}
+            inputBalanceUsd={
+              toUsdRate != null && decimal(formValues.toAmount) ? `${toUsdRate * +formValues.toAmount}` : '0'
```

Comment:
is the '0' really correct here?

Reply:
Yes, tutti wants to show `≈ $0` even if the input is empty. If I'd return `undefined` it would show nothing (which is what we want in case we don't want to show anything)